src/test/integration/configuration-example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "hostname": "https://api.oregonstate.edu",
+  "version": "v1",
+  "api": "teamdynamix",
+
+  "token_api_url": "https://api.oregonstate.edu/oauth2/token",
+  "client_id": "client_id",
+  "client_secret": "client_secret",
+
+  "service_id": "12345"
```

Comment:
Instead of having this service_id be configurable, what are your thoughts on getting the service ID from the main endpoint of the API? You could pick 1 (or a few) random service IDs, and make sure they work properly in the getByID endpoint.

Reply:
Good idea.